airlock/templates/change_file_group.html

```diff
@@ -0,0 +1,44 @@
+{# hide default button, as we are going to autoshow the modal anyway #}
+{% fragment as button %}
+{% endfragment %}
+<style>
+  div.filetype-radio div { display: inline; }
+  div.filetype-radio label { display: inline; }
+</style>
```

Comment:
I don't think these are used (but might make sense to leave them, if we're going to reuse this template for changing filetype)